solidus_social.gemspec

```diff
@@ -39,6 +39,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'solidus_auth_devise'
   spec.add_dependency 'solidus_core', ['>= 2.0.0', '< 4']
   spec.add_dependency 'solidus_support', '~> 0.5'
+  spec.add_dependency 'multi_json'
+  spec.add_dependency 'coffee-rails'
```

Comment:
I can't see coffee files. Maybe there were in the past, but it's no longer the case. Does anything break if we remove the dependency?

Reply:
Yes
The install fails and gives the following error message
```
cannot load such file -- coffee_script (LoadError)
```

Reply:
Probably we can remove this line:

https://github.com/solidusio-contrib/solidus_social/blob/d97cecd7c1f53093fe37d8f47b3ee41e9bbad7f8/lib/solidus_social/engine.rb#L9

Reply:
Yeah, this can probably be removed

Reply:
Removed and fixed the dependency issue here [382d2b](https://github.com/solidusio-contrib/solidus_social/commit/382d2b15f9afb1843bccffb40dbe983539083118)